### PR TITLE
DCS-654 Removing references to prison staff hub and updating README

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -29,8 +29,8 @@ jobs:
       - run:
           name: Run Tests
           command: |
-             set -e
-             npm test -- --runInBand
+              set -e
+              npm test -- --runInBand
```
Comment (Nimphal) [inline .circleci/config.yml:33]: Assume this additional space is intentional?

**Review by Nimphal — DISMISSED** [review]
:boom: 
LGTM!

File: .circleci/config.yml (hunk 1 above)
Comment (andrewrlee) [reply .circleci/config.yml:33]: didn't mean to resolve this! it's just an odd number of characters indent and vs code was complaining

**Review by Nimphal — APPROVED** [review]

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**